LeaderboardBackend/Models/Entities/Run.cs

```diff
@@ -61,13 +61,18 @@ public class Run
 	public List<Participation>? Participations { get; set; }
 
 	/// <summary>
-	/// 	The ID of the `Category` for `Run`.
+	/// 	The ID of the `Category` tied to the `Run`.
 	/// </summary>
 	[Required]
 	public long CategoryId { get; set; }
 
 	/// <summary>
-	/// 	Relationship model for `CategoryId`.
+	/// 	The `Category` object itself, which is tied to the `Run`.
 	/// </summary>
 	public Category? Category { get; set; }
+
+	/// <summary>
+	///     A collection of `NumericalMetric`s scoped to the `Run`.
+	/// </summary>
+	public List<NumericalMetric>? NumericalMetrics { get; set; }
```

Comment:
This seems wrong. Doesn't a run need to be associated with a numerical metric through a value and not the metric itself?

Reply:
It's [EF Core conventions on defining a many-to-many[(https://learn.microsoft.com/en-us/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key#many-to-many). With this, EF Core generates the migration with the appropriate join table and stuff.